.changeset/gentle-spoons-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@openally/mutex": minor
```

Comment:
```suggestion
"@openally/mutex": major
```
For me it's breaking change. @fraxken ?

Reply:
Yes would be better if still used with Node v20

Reply:
Nvm we can still use `free` manually

Reply:
We use `Promise.withResolvers` so this package require v22+ already